recipe/gen_patch_json.py

```diff
@@ -815,7 +815,7 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         if record_name == "iris":
             _rename_dependency(fn, record, "nc_time_axis", "nc-time-axis")
             if record["version"] in iris_updates:
-                record["depends"] = iris_updates[record["version"]]
+                record["depends"].extend(iris_updates[record["version"]])
```

Comment:
You should find the current netcdf pin and replace it like other examples in the file.

Reply:
I was thinking I should do that - but no version pins it until 3.4 (which is coming now, it's in rc0 stage at the moment), and that pins it to !=1.6.1 so am afraid to mess around with the various ways of not pinning/pinning it - wouldn't the append at the end of the deps list do the trick in a safe way?

Reply:
That's fine.

Reply:
yuss! Cheers muchly @beckermr very much appreciated :beer: